--- a/dlls/shell32/shelllink.c
+++ b/dlls/shell32/shelllink.c
@@ -296,33 +296,7 @@
 }
 
 BOOL run_winemenubuilder( const WCHAR *args, BOOL wait )
 {
-    WCHAR app[MAX_PATH];
-    STARTUPINFOW si;
-    PROCESS_INFORMATION pi;
-    BOOL ret;
-    void *redir;
-
-    if (!get_winemenubuilder_path( app )) return FALSE;
-
-    TRACE( "starting %s %s\n", debugstr_w(app), debugstr_w(args) );
-
-    memset( &si, 0, sizeof(si) );
-    si.cb = sizeof(si);
-
-    Wow64DisableWow64FsRedirection( &redir );
-    ret = CreateProcessW( app, (WCHAR *)args, NULL, NULL, FALSE, DETACHED_PROCESS, NULL, NULL, &si, &pi );
-    Wow64RevertWow64FsRedirection( redir );
-
-    if (ret)
-    {
-        if (wait)
-        {
-            wait_for_process( pi.hProcess, INFINITE );
-            GetExitCodeProcess( pi.hProcess, (DWORD *)&ret );
-        }
-        CloseHandle( pi.hProcess );
-        CloseHandle( pi.hThread );
-    }
-    return ret;
+    return TRUE;
 }
 
 static HRESULT WINAPI IPersistFile_fnLoad(IPersistFile* iface, LPCOLESTR pszFileName, DWORD dwMode)
